docs/gettingstarted.md

```diff
@@ -24,30 +24,19 @@ For Linux, there is an [installer script](https://github.com/rbenv/rbenv-install
 
 The following instructions will allow for a global install of rbenv.
 
-1. Install the dependencies for building ruby
-   - centos
-
-         sudo yum install -y openssl-devel readline-devel zlib-devel bzip2 gcc make git
-
-   - ubuntu
-
-         sudo apt update
-         sudo apt install -y libssl-dev libreadline-dev zlib1g-dev bzip2 gcc make git
+1. Install the dependencies for building ruby, referencing the `ruby-build` [Suggested build environment](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment) documentation.
 
 1. clone git repos and setup path
 
-       sudo git clone git://github.com/rbenv/rbenv.git /usr/local/rbenv
-       sudo mkdir /usr/local/rbenv/plugins
-       sudo git clone git://github.com/rbenv/ruby-build.git /usr/local/rbenv/plugins/ruby-build
-       sudo tee /etc/profile.d/rbenv.sh <<< 'export PATH="/usr/local/rbenv/plugins/ruby-build/bin:/usr/local/rbenv/bin:$PATH"'
-       sudo tee -a /etc/profile.d/rbenv.sh <<< 'source <(rbenv init -)'
-
-   now the shell needs to be restarted so that the rbenv commands are in the path, and then `rbenv init -` adds the shims path to the PATH as well.
+   - [Basic GitHub Checkout/setup of rbenv](https://github.com/rbenv/rbenv#basic-github-checkout)
+   - [Installation of `ruby-build`](https://github.com/rbenv/ruby-build#installation)
 
 1. Install ruby and set it as the version to use by default
 
-       rbenv install 2.6.3
-       rbenv global 2.6.3
+       rbenv install 2.6.7
+       rbenv local 2.6.7
```

Comment:
We instruct them to use 2.6.3 because that's what we use in salt.  We should probably make that more clear.  They should be able to use a fairly wide range of versions without much difficulty I'm pretty sure.

Reply:
Ah, sounds good. I've added back `2.6.3`, and a note, per our usage of the Ruby version referenced in our CI:
- https://github.com/saltstack/salt-jenkins-pipeline-libs/blob/master-1.10/vars/runTests.groovy#L44